recipes/libxcrypt/all/conanfile.py

```diff
@@ -43,7 +43,7 @@ def source(self):
         os.rename("{}-{}".format(self.name, self.version), self._source_subfolder)
 
     def build_requirements(self):
-        self.build_requires("automake/1.16.2")
+        self.build_requires("automake/1.16.3")
         self.build_requires("libtool/2.4.6")
```

Comment:
Because libtool is also a build requirement, automake a dependency of libtool, maybe we could remove automake here.

Reply:
@madebr what do you think about this?

Reply:
That can be done.
`automake` should only be added explicitly when using the `compile` or `ar-lib` scripts.